Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
I don't think the FORCE_INLINE is really necessary too. By being defined inside the class they are by default inline and frankly I can only see 3 things being exposed in the object files.
```
00009488 T _Z16mesh_buffer_linefffffRKhhh
00005ffe W _ZN17mesh_bed_leveling5get_zEff
0000c0c0 T _ZN17mesh_bed_leveling5resetEv
```
And you can see that `get_z()` did not get to be inlined, which I imagine came because there is probably a cut of size the compiler compares to. By forcing that to be inline the binary grew with 3366 bytes. Removing the other FORCE_INLINE/static didn't affect the binary at all and no new "mesh" methods got exposed in the object files.